worldedit-core/src/main/java/com/sk89q/worldedit/entity/Player.java

```diff
@@ -470,13 +470,21 @@ default void loadClipboardFromDisk() {
                     }
                 } catch (EmptyClipboardException ignored) {
                 }
-                DiskOptimizedClipboard doc = new DiskOptimizedClipboard(file);
+                DiskOptimizedClipboard doc;
```

Comment:
I think it would be cleaner to have a static factory method in DiskOptimizedClipboard that handles loading instead of having to catch the exception here.